opg-php-fpm-71-ppa-1604/Dockerfile

```diff
@@ -4,7 +4,6 @@ ADD  docker/confd /etc/confd
 ADD  docker/logrotate.d/app /etc/logrotate.d/app
 ADD  docker/my_init.d /etc/my_init.d
 ADD  docker/nginx /etc/nginx
-ADD  docker/service/php-fpm-stats /etc/sv/php-fpm-stats
 ADD  docker/service/php-fpm /etc/sv/php-fpm
```

Comment:
Change ADD to COPY ?

Reply:
Outside the scope of this ticket. Agreed this isn't ideal, or what we actually want to do.